src/Service/S3/src/Result/GetObjectOutput.php

```diff
@@ -454,7 +454,7 @@ protected function populateResult(Response $response): void
         $this->Metadata = [];
         foreach ($headers as $name => $value) {
             if ('x-amz-meta-' === substr($name, 0, 11)) {
-                $this->Metadata[$name] = $value[0];
+                $this->Metadata[substr($name, 11)] = $value[0];
```

Comment:
Could the user manually add metadata in the S3 dashboard. Would that be prefixed here?

Reply:
I'm not sure to understand what you mean. 

note: Aws SDK removes the prefix as well

Reply:
I meant: Will `$name` always include the prefix? No matter how the metadata was added?

Reply:
prefix it checked at line 456

Reply:
Oh, sorry. I didnt see that.